### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by tdphut** [tdphut/sites#2]
File [index.html](https://github.com/tdphut/sites/blob/master/huddle-landing-page-945317/index.html)

* Missing content for `alt` attribute for `header` and `footer`.
* Missing `aria-hidden` attribute for parent's icon elements.
* Missing `ara-label` attribute for social icon links.

**Comment by tdphut** [comment]
Fixed issue #2
